### PR TITLE
Add Readme section for 3rd party software

### LINKED ISSUE #610: WP.ORG: Undocumented use of a 3rd Party or external service
We permit plugins to require the use of 3rd party (i.e. external) services, provided they are properly documented in a clear manner.

We require plugins that reach out to other services to disclose this, in clear and plain language, so users are aware of where data is being sent. This allows them to ensure that any legal issues with data transmissions are covered. This is true even if you are the 3rd party service.

In order to do so, you must update your readme to do the following:
Clearly explain that your plugin is relying on a 3rd party as a service and under what circumstances
Provide a link to the service .
Provide a link to the service terms of use and/or privacy policies.
Remember, this is for your own legal protection. Use of services must be upfront and well documented.

Example(s) from your plugin:
# Domain(s) not mentioned in the readme file.
21_21-26-52_gatherpress/includes/core/classes/class-event.php:812 $args = array(
'BEGIN:VCALENDAR',
'VERSION:2.0',
'PRODID:-//GatherPress//RemoteApi//EN',
'BEGIN:VEVENT',
sprintf( 'URL:%s', esc_url_raw( get_permalink( $this->event->ID ) ) ),
sprintf( 'DTSTART:%s', sanitize_text_field( $datetime_start ) ),
sprintf( 'DTEND:%s', sanitize_text_field( $datetime_end ) ),
sprintf( 'DTSTAMP:%s', sanitize_text_field( $datetime_stamp ) ),
sprintf( 'SUMMARY:%s', sanitize_text_field( $this->event->post_title ) ),
sprintf( 'DESCRIPTION:%s', sanitize_text_field( $description ) ),
sprintf( 'LOCATION:%s', sanitize_text_field( $location ) ),
'UID:gatherpress_' . intval( $this->event->ID ),
'END:VEVENT',
'END:VCALENDAR',
);

### PR DESCRIPTION
**Pull request by jmarx**
### Description of the Change
Add a section in the readme dedicated to giving credit to third party software integrations.

Closes #610
Can be added to @carstingaxion  #653 
I also created a pr off of @carstingaxion 's  branch:
https://github.com/GatherPress/gatherpress/pull/657 (closed for now

### Credits

Props @carstingaxion , @mauteri

### CODE REVIEW

**Review by stephenerdelyi — CHANGES_REQUESTED** [review]
Do we need to add the following?
- [Google Maps](https://developers.google.com/maps): Used to render maps - only when Google Maps is selected as your mapping platform.
- [Open Street Map](https://www.openstreetmap.org/): Used to render maps - only when Open Street Map is selected as your mapping platform.

**Comment by jmarx** [comment]
> Do we need to add the following?
> 
> * [Google Maps](https://developers.google.com/maps): Used to render maps - only when Google Maps is selected as your mapping platform.
> * [Open Street Map](https://www.openstreetmap.org/): Used to render maps - only when Open Street Map is selected as your mapping platform.

@stephenerdelyi  I was going to add OSM as part of our OSM feature. I'm not sure if we need to add google maps since we are just iframing their map. What do you think @mauteri ? Also, does anyone know why the a php unit test would be failing on a simple readme change?

**Comment by carstingaxion** [comment]
> Also, does anyone know why the a php unit test would be failing on a simple readme change?

@jmarx the logs know the answer, "they" said:

> Could not connect to deb.debian.org:80 (146.75.94.132), connection timed out

Should probably work on the next run.

**Comment by jmarx** [comment]
@mauteri Are we good to merge this? Or should we add the map items?

**Comment by carstingaxion** [comment]
I added all the changes also to #653 .

File: README.md [hunk 1]
```diff
@@ -214,3 +214,13 @@ For more info on wp-env package, consult the [Block Handbook's page](https://dev
...
+
+## Third-Party Libraries
+
+This plugin leverages the following third-party libraries for various functionalities:
+
+- [Moment.js](https://momentjs.com/): Used for date and time manipulation.
+- [Moment-Timezone](https://momentjs.com/timezone/): Adds timezone support to Moment.js.
+- [Playwright](https://playwright.dev/): Enables browser automation for testing and interaction.
```
Comment (mauteri) [inline README.md:224]: Not needed here as it is not packaged with the plugin

**Review by mauteri — CHANGES_REQUESTED** [review]

**Review by mauteri — CHANGES_REQUESTED** [review]

**Comment by mauteri** [comment]
Closing this PR as this should be done in @carstingaxion's PR at this point.

**Pull request closed without merging**